soroban-env-host/benches/common/measure.rs

```diff
@@ -127,31 +137,32 @@ impl Measurements {
     }
 
     pub fn report_table(&self) {
+        // data must be preprocessed
+        assert_eq!(self.measurements.len(), self.net_meas_per_iter.len());
+
         use std::io::Write;
         use thousands::Separable;
         let mut tw = TabWriter::new(vec![])
             .padding(5)
             .alignment(Alignment::Right);
 
-        writeln!(
-            &mut tw,
-            "input\tinput_baseline\tcpu_insns\tcpu_baseline\tinsns/input\tmem_bytes\tmem_baseline\tbytes/input\ttime nsecs\tnsecs/input"
-        )
-        .unwrap();
-        for m in self.measurements.iter() {
+        writeln!(&mut tw, "iterations\tinputs\tcpu_insns_total\tcpu_insns_base\tmem_bytes_total\tmem_bytes_base\ttime_ns_total\ttime_ns_base\tinput/iter\tnet_insns/iter\tnet_bytes/iter\tnet_ns/input").unwrap();
```

Comment:
it seems to me this is missing net_insns/input which is what the model-fit will report below -- might be nice to eyeball that?

Reply:
Ok, will add that.

Reply:
I have added what you suggested and also renamed some column names e.g. "net_insns/iter" --> "ave_net_insns" to make it cleaner.